api/models/user.go

```diff
@@ -208,3 +210,73 @@ func AddItemsToUserInventory(userId primitive.ObjectID, items []interfaces.GymRe
 
 	return nil
 }
+
+// GetLoomiesByUser returns an array of loomies according with user
+func GetLoomiesByUser(loomiesArray []primitive.ObjectID) ([]interfaces.UserLoomiesRes, error) {
```

Comment:
Maybe you can change the name to `GetLoomiesByIds`, the user isn't an argument of the function.